### DISCUSSION TITLE
More internalLoadbalancers

### DISCUSSION THREAD

**Discussion by AkakievKD** [kubesphere/kubekey#1321] [Ideas]
Will you be adding more internalLoadbalancers to choose any internalLoadbalancers other than haproxy?

**Comment by 24sama** [comment]
Hi @Nello-Angelo 
We plan to add the nginx, but it's a lower priority because we're refactoring the kk operator mode. If you are interested in it, welcome to create a PR!
You can refer to the [kubespray](https://github.com/kubernetes-sigs/kubespray/tree/master/roles/kubernetes/node/templates/loadbalancer)

**Reply by AkakievKD** [reply]
I don't like the kubespray))
Okay, no problem, thanks, i just wanted to know about your plans.

**Comment by AkakievKD** [comment]
OK, no problem. I'll try to do it when I'm free. Thanks for this.

ср, 8 июн. 2022 г. в 18:31, Leo Li ***@***.***>:
